src/pages/settings/bank-accounts/common/validation/ValidationInterface.tsx

```diff
@@ -0,0 +1,13 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+export interface BankAccountValidation {
+  bank_account_name?: string[];
```

Comment:
You can put the validation interface in the component itself. No need for separate file.
Also, it returns pure TS, so correct extension to `ts` (instead of `tsx`).